api/src/main/java/org/openmrs/logic/LogicCriteria.java

```diff
@@ -13,11 +13,11 @@
 import java.util.Date;
 import java.util.Map;
 
-import org.hibernate.criterion.Distinct;
 import org.openmrs.logic.op.And;
 import org.openmrs.logic.op.AsOf;
 import org.openmrs.logic.op.Average;
 import org.openmrs.logic.op.Count;
+import org.openmrs.logic.op.Distinct;
```

Comment:
Is this used anywhere in this class?

Reply:
It is used here in the docstrings, I think it was originally using criterion Distinct instead of logic.op like the others:

	/**
	 * Apply the {@link Distinct} operator to the LogicCriteria
	 * 
	 * @return LogicCriteria with Distinct operator in it
	 * @see org.openmrs.logic.op.Distinct
	 */
	public LogicCriteria distinct();